pages/success/index.tsx

```diff
@@ -230,13 +149,21 @@ const RightSide = styled(Flex)(
   })
 );
 
-const StyledCopyright = styled("div")(
+const StyledUL = styled("ul")(
+  css({
+    listStyle: "none",
+    m: 0,
+    pl: 0,
+  })
+);
+
+const TextFlex = styled(Flex)(
   css({
-    color: "rgba(255, 255, 255, 0.8)",
-    display: ["none", "block"],
-    fontSize: "text-sm",
-    textAlign: "center",
-    textTransform: "uppercase",
+    alignItems: "stretch",
+    flexDirection: "column",
+    lineHeight: "xl",
+    minHeight: 6,
+    minWidth: ["auto", "304px"],
```

Comment:
```suggestion
    minWidth: ["auto", "304px"],
        h3: {
      fontSize: "text-xl",
      lineHeight: ["md", "lg"],
      my: 0,
      pt: [5, 6],
    }, 
        ul: {
      listStyle: "none",
      m: 0,
      pl: 0,
    },
    li: {
      display: "flex",
      lineHeight: "md",
      pt: [2, 4],
      "&:nth-child(1)": {
        pt: [2, 3],
      },
    }
```
Add the h3 styles into the parent container rather, then delete `StyledH3`. Those H3s can then just have a `<h3>` tag. Then do the same for `StyledUL` and `StyledLi`.